src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download %s</string>
+  <string name="error_syncing_format">Could not sync %s</string>
+  <string name="error_validating_format">Could not validate %s</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">Downloaded %s</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
   <string name="ignore_all_updates">Ignore all updates</string>
   <string name="ignore_this_update">Ignore this update</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">Older than the version installed on your device.
+    Uninstall the current application to install the older version.</string>
   <string name="incompatible_platforms_description_format">Your %1$s platform is not supported.
     Supported platforms: %2$s.</string>
   <string name="incompatible_sdk_description_format">Your %1$s (SDK version %2$d) is not supported. %3$s</string>
-  <string name="incompatible_sdk_max_description_format">Maximum SDK version is %d.</string>
-  <string name="incompatible_sdk_min_description_format">Minimum SDK version is %d.</string>
+  <string name="incompatible_sdk_max_description_format">No SDK version above %d is allowed.</string>
+  <string name="incompatible_sdk_min_description_format">No SDK version below %d is allowed.</string>
```

Comment:
(old conversation removed from this change)



> It's better to use "SDK version" instead of "SDK" (because "SDK version" is already used in `incompatible_sdk_description_format`).
> "SDK version %d is the latest allowed version" is good as "Maximum SDK version is %d" replacement. But what about "Minimum SDK version is %d"?

"No SDK version above %s is allowed" doesn't make any sense, because SDK version can't be __allowed__ in this context.

I still think that it's not possible to describe these incompatibility problems as "Maximum SDK version is %d." and "Minimum SDK version is %d." Are you sure it's even necessary to change them?

Reply:
It doens't make sense because no such version exists? Even if it did, it is a hard limit (?) It doesn't need to be changed, I just thought the change makes it easier to grasp.

Reply:
Think about "SDK version 24" as about "Android 7.0", so instead of "No SDK version above 24 is allowed" it will become "No Android above 7.0 is allowed". My sense of English tells me that such attitude is just weird, and "allowed" here is not a correct word to describe what happens.

Reply:
I thought the Android being tied to SDK version was already thrown out.
Is it OK as it is now, or revert? (It would be "beyond" in your example)

Reply:
How is it now? "No SDK version above %d is allowed." definitely not OK.

I'd preferred to revert both lines to initial "Maximum SDK version is %d." and "Minimum SDK version is %d.".

Reply:
Hmm, I don't understand, revert to which point?

I'd preferred to keep these 2 lines as they were initially, they're OK to me, and I actually don't see any improvements in changing them.